src/compounds/tabs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@uswitch/trustyle.tabs",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "ts:main": "src/index.tsx",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "--------------- BUILDING": "----------------------",
+    "clean": "rm -rf lib",
+    "build": "npm run clean && tsc"
+  },
+  "peerDependencies": {
+    "@emotion/core": "^10.0.27",
+    "react": "^16.7.0"
+  },
+  "dependencies": {
+    "@uswitch/trustyle.flex-grid": "^2.2.2",
+    "react-indiana-drag-scroll": "^1.6.1"
```

Comment:
the browser will need to parse 8kb of JS for this, is there any smaller libraries

Reply:
Where did you get that number from? It seems to be 4.7kb before minifying: https://unpkg.com/react-indiana-drag-scroll@1.6.1/dist/index.js